src/CommandLine/Endpoint.cs

```diff
@@ -8,7 +8,7 @@
 
     static class Endpoint
     {
-        public static async Task Create(ServiceBusAdministrationClient client, CommandArgument name, CommandOption topicName, CommandOption subscriptionName, CommandOption<int> size, CommandOption partitioning)
+        public static async Task Create(ServiceBusAdministrationClient client, CommandArgument name, CommandOption topicName, CommandOption topicToPublishTo, CommandOption topicToSubscribeOn, CommandOption subscriptionName, CommandOption<int> size, CommandOption partitioning)
```

Comment:
Given that TopicName is being marked as Obsolete in AzureServiceBus transport, is it worth having two different overloads here - one with just topic name (marked as obsolete) and one with publish/subscribe topics?  The thought being that once TopicName is removed from the transport, it will be easier to remove the corresponding functionality here.

Reply:
@lailabougria Andreas and I had a conversation about this. I told him the idea of this change is to make sure the existing `topicName` option can still be used and represents the `single` topic topology while the other two parameters represent the hierarchy options. Validation will take care those parameters can only be used in valid combinations. Once validation is over I could theoretically map the single topic name parameter to the publish/subscribe topics but then I would create an artificial state of these parameters that validation previously made sure it should not be possible which also made me a bit iffy. I'm not opposed changing this code if we can come up with a better solution but I'm also not too overly worried if we leave it like that since those API parameter closely follow the assumptions of the command line usage anyway.